xml/security_firewall.xml

```diff
@@ -961,7 +961,174 @@ nfs-rpc
      <command>firewall-rpc-helper.py --help</command>.
     </para>
    </sect3>
-  </sect2>
+   <sect3 xml:id="sec-security-firewall-firewalld-nftables">
```

Comment:
I recommend collecting all nftables-related subsections under this subsection, i.e. this is `sect3`, and all the rest pertaining to nftabes (up to More information, incl.) should be `sect4`.

Reply:
done